Java/src/test/java/com/gildedrose/GildedRoseTest.java

```diff
@@ -7,11 +7,85 @@
 class GildedRoseTest {
 
     @Test
-    void foo() {
-        Item[] items = new Item[] { new Item("foo", 0, 0) };
+    void testNormalDegrade() {
+        Item testItem = new Item("test_item", 3, 10);
```

Comment:
we don't need to make new instances in each method, we should have a method to reuse these